### ISSUE TITLE
Allow gcp-filestore-backups to support multiple filestores?

### ISSUE THREAD

**Issue by sgibson91** [2i2c-org/infrastructure#4449]
https://github.com/2i2c-org/infrastructure/blob/main/helm-charts/images/gcp-filestore-backups/gcp-filestore-backups.py was written to only accept a single name of a filestore instance to create backups for. However, in our terraform code we do actually permit deployment of multiple filestores (if only for the case to functionally permit resizing a GCP Filestore down). Is there a use case for gcp-filestore-backups to also backup multiple filestores in the same project?

**Comment by yuvipanda** [comment]
@sgibson91 yes, you should implement that! Do you need any more information than what you have? If this feels refined enough, please put this card in refined as part of the epic you're working on, and pick it up when you're done with the existing tasks there!

Thanks for catching this.
